lib/routes/common/feature.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+const { Router } = require('express');
+const version = 1;
+
+const legacyFeatureMapper = require('../../data-helper/legacy-feature-mapper');
+
+exports.router = config => {
+    const { featureToggleStore } = config.stores;
+    const router = Router(); // eslint-disable-line new-cap
+
+    router.get('/', (req, res) => {
```

Comment:
I feel it is cheaper to duplicate this than to share it. E.g. we don't need the legacy mapper in admin, because this is only needed to support old clients.